### PR TITLE
Oct dev update

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: content/posts/irods-development-update-october-2023.md [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+
+The iRODS C++ S3 API is coming together very well. Support for specifying byte ranges via the range header has been added. The **ListBuckets** API operation has been implemented as well.
+
+The effort around OpenID Connect has finally landed in the iRODS HTTP API! This is a major step for iRODS, but there's still a lot of work to do. Aside from that, we've made the HTTP API compatible with iRODS 4.2.11 and 4.2.12. A compatibility option was added which, when enabled, instructs the iRODS HTTP API to only use 4.2.11+ API calls.
+
+Python iRODS Client 1.1.9 is very close. The work for client-side redirection and auto-closing of data objects is mostly complete. All that's left is testing and documentation. To add to that, the PRC will soon give users a way to adjust the behavior of the library using a local settings file.
```
Comment (trel) [inline content/posts/irods-development-update-october-2023.md:20]: config file for library feels weird.  let's hold on this part until discush.

Comment (trel) [reply content/posts/irods-development-update-october-2023.md:20]: it's fine - resolving.

**Comment by trel** [comment]
please squash

**Comment by korydraughn** [comment]
Squashed and ready for merging.

**Pull request merged**